components/searchBar/searchBar.tsx

```diff
@@ -1,19 +1,49 @@
-import React from 'react';
+import { countReset } from 'console';
+import React, { useState } from 'react';
+
+// Components
+import SearchSVG from '../svg/searchSVG';
 
 interface iSearchbar {
   placeholder: string;
+  value?: string;
+  setValue?: (e: any) => void;
```

Comment:
Avoid using any for events

Reply:
Thank you so much for your suggestion, Now I made the changes for interfaces and in the functions passing with the events for react and typescript 

```
interface iSearchbar {
  placeholder: string;
  value: string;
  setValue: (e: string) => void;
}

const searchBar: React.FC<iSearchbar> = ({ placeholder, value, setValue }) => {
  const handleChange = (e: ChangeEvent<HTMLButtonElement>) => {
    setValue(e.target.value);
  };

  const handleClick = (e: FormEvent) => {
    e.preventDefault();
...
...
```